Spring-server/src/main/java/hello/LoadDatabase.java

```diff
@@ -0,0 +1,20 @@
+package hello;
+
+import org.springframework.boot.CommandLineRunner;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+class LoadDatabase {
+
+    @Bean
+    CommandLineRunner initDatabase(UserRepository repository) {
+        return args -> {
+            repository.save(new User("sample", "pass",
+                    "sample@gmail.com", "qwerty",
+                    "smtp.google.com", 883,
+                    "imap.google.com", 666));
```

Comment:
For the visibility i'd like to change "google.com" on "rambler.ru", for example, 'cause we don't have the Gmail-support for now :c